Mathlib/Tactic/Have.lean

```diff
@@ -12,6 +12,9 @@ import Lean.Meta.Tactic.Assert
 
 This file extends the `have`, `let` and `suffices` tactics to allow the addition of hypotheses to
 the context without requiring their proofs to be provided immediately.
+
+As a style choice, this should not be used in mathlib; but is provided for downstream users who
+preferred the old style.
```

Comment:
Could you add the same comment to `Tactic.Replace`?

Reply:
Done